.travis.yml

```diff
@@ -54,3 +56,12 @@ jobs:
         on:
           tags: true
           go: 1.11.x
+    - name: 'Deploy to staging'
+      stage: staging
+      script:
+        - make docker-push
```

Comment:
do we need `make packages` here? looks like we do because the prev stage is skipped